utils/flakiness-dashboard/FlakinessDashboard.js

```diff
@@ -84,19 +94,31 @@ class Dashboard {
       throw new Error('cannot parse dashboard data: missing "version" field!');
     if (data.version > DASHBOARD_VERSION)
       throw new Error('cannot manage dashboards that are newer then this');
-    const builds = data.builds.map(build => new Build(build.timestamp, build.name, build.url, build.tests));
+    const builds = data.builds.map(build => new Build(build.ts, build.n, build.u, build.t.map(test => ({
+      testId: test.i,
+      name: test.n,
+      description: test.d,
+      url: test.u,
+      result: test.r,
+    }))));
     return new Dashboard(name, dashboardPath, builds, options);
   }
 
   async saveJSON() {
     const data = { version: DASHBOARD_VERSION };
     data.builds = this._builds.map(build => ({
-      timestamp: build._timestamp,
```

Comment:
Uh, git compresses things. I think this is unnecessary.

Reply:
that's for the custom flakienss viewer. This saves ~100kb in my tests.

Reply:
How are you operating where an uncompressed 100kb matters? I'd much rather have a readable data format.

Reply:
> uncompressed 100kb matters

Compressed 100kb